scripts/schema/subset_filter.py

```diff
@@ -50,29 +49,56 @@ def warn(message):
     print(message)
 
 
+ecs_options = ['fields', 'enabled', 'index']
```

Comment:
So would idea here be that we could set `enabled` and `index` per subset if we wanted instead of having to do it globally in the custom schema files for example like we are doing here: https://github.com/elastic/endpoint-package/blob/master/custom_schemas/custom_endpoint.yml#L28

Reply:
Yeah, and when merging subsets together the field is enabled in the index if it is enabled in any of the subsets. This also lets us easily disable indexing on ECS fields.